turbo/templatetags/turbo_streams.py

```diff
@@ -15,8 +15,8 @@ def turbo_stream_from(model_instance):
     return {"signed_channel_name": signed_channel_name}
 
 
-@register.inclusion_tag("turbo/turbo_stream_id.html")
+@register.simple_tag
```

Comment:
Rendering a template seemed unneeded here? Seems like it's just returning a string.

Reply:
Ah yep. Good catch 😄